resources/js/Components/Login/LoginForm/index.tsx

```diff
@@ -0,0 +1,93 @@
+import React, { useState, useCallback } from "react"
+import InputText from "@/Components/InputText";
+import UserIcon from "/public/assets/images/icon/user-2.svg"
+import EmailIcon from "/public/assets/images/icon/email-2.svg"
+//Formik
+import { Formik, FormikValues } from "formik";
+import { loginSchema } from "@/Validations/loginValidation";
+import InputPassword from "@/Components/InputPassword";
+import InputCheckBox from "@/Components/InputCheckBox";
+
+
+
+const LoginForm = () => {
+	const [email, setEmail] = useState("");
+	const [password, setPassword] = useState("");
+	const [passwordType, setPasswordType] = useState("password");
+	const [passwordIconClass, setPasswordIconClass] = useState("bi bi-eye-slash")
+
+	const onSubmitFormik = useCallback((values: FormikValues) => {
+		console.log(values)
+	}, [])
+
+	const onChangeEmail = (event: React.ChangeEvent<HTMLInputElement>): void => {
+		const value = event.target.value;
+		setEmail(value);
+	}
+
+	const onChangePassword = (event: React.ChangeEvent<HTMLInputElement>): void => {
+		const value = event.target.value;
+		setPassword(value);
+	}
+
+	return (
+		<>
```

Comment:
indents with 2 spaces, not tabs

Reply:
For some reason, my IDE has 2 spaces, not tabs, but GitHub shows two tabs. 
I will try to fix it somehow